zurb-template/src/data/event.json

```diff
@@ -15,18 +15,18 @@
 },
 
 { 
-"text": "Set single or multiple runs for task with count parameter", 
-"link": "https://github.com/intelsdi-x/snap/blob/master/docs/TASKS.md#task-manifest" 
+"text": "2.0 release introduces a new plugin type: streaming collectors", 
+"link": "https://github.com/intelsdi-x/snap/blob/master/docs/STREAMING.md" 
 },
 
 { 
-"text": "Plugin Authors: Go RPC now deprecated", 
-"link": "https://github.com/intelsdi-x/snap/issues/1289"
+"text": "Plugin Authors: \"Supported\" plugins tag indicates premier plugins", 
+"link": "https://github.com/intelsdi-x/snap/blob/docs/PLUGIN_STATUS.md"
```

Comment:
This link also does not work. You should change the address to https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_STATUS.md